python/graphscope/config.py

```diff
@@ -77,4 +77,12 @@ class GSConfig(object):
     show_log = False
     log_level = "INFO"
 
+    # GIE engine params
+    engine_params = None
+
+    # GIE instance will be created automatically when a property graph loaded.
+    # Otherwise, you should create a GIE instance manually by `sess.gremlin` if
+    # `initializing_interactive_engine` is False
+    initializing_interactive_engine = True
+
```

Comment:
I'm wondering if the `GSConfig` docs could be rendered in sphinx properly.

Reply:
We didn't expose `GSConfig` in Doc. Instead, `graphscope.set_option` and `graphscope.get_option` exposed.

https://graphscope.io/docs/reference/generated/graphscope.set_option.html#graphscope.set_option